src/dpsvalidationservice/dfcmsords.json

```diff
@@ -0,0 +1,87 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "ORDS generated API for web",
+        "version": "1.0.0"
+    },
+    "host": "localhost",
+    "basePath": "/ords/deva/dfcmsords/web",
```

Comment:
Question . Is the base path going to different when environment change ?

Reply:
Yes it will, if you look at the ords client there is dpsvalidationservice.ords.dfcms.client.base-url that overwrite the value when we bootstrap the app.